test/functional/page_objects/visual_builder_page.js

```diff
@@ -82,6 +83,64 @@ export function VisualBuilderPageProvider({ getService, getPageObjects }) {
       await testSubjects.click('markdownDataBtn');
     }
 
+    /**
+     *
+     * getting all markdown variables list which located on `table` section
+     *
+     * **Note**: if `table` not have variables, return `No variables available for the selected data metrics.`
+     * @returns {Promise<string[] | string>}
```

Comment:
It is better to return `[{name, value}]`
Don't try to implement universal function, for error msg use another one :)